glossary.html

```diff
@@ -34,8 +34,8 @@
 <div class="width-wrap">
     <header id="top">
         <nav>
-        <label class="burger" for="burger-toggle"></label>
             <input type="checkbox" id="burger-toggle" name="burger-toggle">
+            <label class="burger" for="burger-toggle"></label>
```

Comment:
You forgot to change the order of the label and the checkbox on your other pages